.github/workflows/monitor.yml

```diff
@@ -0,0 +1,32 @@
+name: Monitor
+
+# Controls when the workflow will run
+on:
+  # Triggers the workflow on push events but only for the "main" branch
+  push:
+    branches: ['main']
+
+# A workflow run is made up of one or more jobs that can run sequentially or in parallel
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
```

Comment:
This is that additional workflow you added during our group coding session. Is there anywhere I can read to understand how this file gets picked up by Github, reads it and like, figures out how to do all of this for us?

Reply:
This [documentation](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions) covers most of those questions, I believe, or at least links to other docs that do 😄